### DISCUSSION TITLE
Is there a FontBuilder like FontSizeBuilder?

### DISCUSSION THREAD

**Discussion by LeonardoIPM** [eclipse-birt/birt#1641] [Q&A]
I'm creating a new plugin to add a new ReportItem, struggling with it... Is there a documentation to help me to implement new plugins? 

Edit: I resolved my issues and found that doesn't exist any updated documentation :(

**Comment by hvbtup** [comment]
Then maybe you should share your findings to improve the situation...

**Reply by LeonardoIPM** [reply]
I created new Control classes to handle with fontFamily and fontSize by my own. 
To fontFamily I used GraphicsEnvironment.getLocalGraphicsEnvironment().getAvailableFontFamilyNames() to create a comboBox and setting the value on fontFamily property, getting this property when rendering the report.
